packages/uswds-core/src/styles/mixins/typography/typeset.scss

```diff
@@ -55,10 +51,6 @@ Sets:
   margin-top: 0;
   max-width: measure($theme-text-measure);
 
-  * + & {
-    margin-top: 1em; // TODO: add as var to settings?
-  }
-
```

Comment:
`* + &` gets confused when added to mixins, so moved these items directly inside the style rules.